### PR TITLE
Remove actions query parameters from docs

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

**Comment by helfi92** [comment]
@djmitche I'm trying to see if we can merge this today if possible. We don't want people to start planning on the older design of queries.

File: docs/actions.md [hunk 1]
```diff
@@ -18,42 +18,47 @@ An action is comprised with the following properties:
...
 
 ## How to trigger an action
-To trigger an action, make a `POST` request to a URL constructed from the url field with query
-arguments added based on the `context`.
+To trigger an action, use the action's `url` property to make a `POST` request to it.
+Depending on the action's `context`, substitute the following parameters in the `url`:
+
+| `context`   | Path parameters                                      |
+|-------------|------------------------------------------------------|
+| provisioner | :provisionerId                                       |
```
Comment (jonasfj) [inline docs/actions.md]: Can we use `<provisionerId>`  like we do in `entries[i].route` in `reference.json` files, see docs:
https://docs.taskcluster.net/manual/design/apis/reference-format

**Comment by djmitche** [comment]
+1 with jonas's suggestion

**Comment by helfi92** [comment]
Done! Will merge later today.

**Pull request merged**